### PR TITLE
Dev - add faq page, revise navigation as needed

### PR DESCRIPTION
**Pull request by izhanchang**
Create FAQ page and revise navigation page. Also updated Equipment with 6 esprites and fixed Services typos

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -22,7 +22,8 @@
               <ul class="dropdown-menu dropdown-menu-dark" aria-labelledby="header-nav-whatWeDo-dropdown">
                 <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'services') %} active" aria-current="page"{% else %}"{% endif %} href="{{ '/services/' | relative_url }}">Services</a></li>
                 <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'equipment') %} active" aria-current="page"{% else %}"{% endif %}  href="{{ '/equipment/' | relative_url }}">Equipment</a></li>
-                <li><hr class="dropdown-divider"></li>
+                <li><a class="dropdown-item{% if (page contains 'nav_page_specific' and page.nav_page_specific == 'FAQ') %} active" aria-current="page"{% else %}"{% endif %}  href="{{ '/FAQ/' | relative_url }}">FAQ</a></li>
```
Comment (DaAwesomeP) [inline _includes/nav.html:25]: `page.nav_page_specific == 'FAQ` should match the `nav_page_specific` in `faq.md`. So it should change to `faq` (all lowercase).

Comment (DaAwesomeP) [inline _includes/nav.html:25]: URLs should be all lowercase, change to `/faq/`.

File: pages/faq.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: FAQ
+permalink: /FAQ/
```
Comment (DaAwesomeP) [inline pages/faq.md:4]: URLs should be all lowercase, change to `/faq/`.

File: pages/faq.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+nav_page: whatWeDo
+nav_page_specific: faq
+---
+
+# Frequently Asked Questions
+
+<hr class="bg-primary"/>
+
+Have a question about AB Tech, getting involved, or hiring us? We have answers (well, most of the time). Don't see your question below? Please email <a href = "mailto: abtech@andrew.cmu.edu">abtech@andrew.cmu.edu</a> with any questions you may have.
```
Comment (DaAwesomeP) [inline pages/faq.md:13]: Change link to be `<a href="mailto:abtech@andrew.cmu.edu" class="user-select-all">abtech@andrew.cmu.edu</a>`. This adds a helper class that makes selecting it a bit easier.

File: pages/faq.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+- <a href="#General">General</a>
+- <a href="#Getting Involved">Getting Involved</a>
+- <a href="#Requesting AB Tech">Requesting AB Tech</a>
+
+<h2>
```
Comment (DaAwesomeP) [inline pages/faq.md:19]: Change this `h2` block to just be `## General`. You should be able to link to it as `#general`.

File: pages/faq.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+</ol>
+Techroom will be on your left after passing the double doors.
+
+##### Who works AB Tech events?
+Students! Affectionately known as techies, our members are undergraduate and graduate students studying anything from engineering to computer science to business to art to English, and everything in between. Occasionally, alumni will also return to help with our large events.
+
+##### How is AB Tech different from technical theatre or Scotch ‘n’ Soda?
+While we do partner and share equipment with the drama department and Scotch ‘n’ Soda (in fact, some of our members are also involved with S’n’S), as the university’s production organization, we support a wider range of events. We specifically specialize in event production with handling entertainment technology; however, you can expect some overlap in knowledge between us and S’n’S.
+
+<h2>
```
Comment (DaAwesomeP) [inline pages/faq.md:44]: Change this h2 block to just be `## Getting Involved`. You should be able to link to it as `#getting-involved`.

File: pages/faq.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+##### What else do techies do?
+Sure, we work hard, but we also have a lot of fun together. After large events, we often get together for a celebratory meal off campus. We’ve also hosted game nights, karaoke, Fuku runs and ski trips for our members. If you’ve got an idea for a social event, let us know! 
+
+##### How do I get involved?
+Come talk to us! We are present at the FAIR and Winter FAIR - just look for the bright lights and follow the loud music. You can also join the mailing list by hitting the <a href="/join">Join</a> button above. See a person in a tech shirt? Ask them about the next event you can help at and they should hopefully be able to point you in the right direction.
+
+##### How else can I get connected?
+Join our <a href="https://www.facebook.com/cmuabtech" target="_blank">Facebook</a> page and follow us on <a href="https://www.instagram.com/cmuabtech/" target="_blank">Instagram</a> @cmuabtech to see all the cool things we’re doing!
+
+<h2>
```
Comment (DaAwesomeP) [inline pages/faq.md:81]: Change this h2 block to just be `## Requesting AB Tech`. You should be able to link to it as `#requesting-ab-tech`.

File: pages/faq.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+- Some types of events favor certain venues over others. Before reserving a venue, please contact us with your event details and we can suggest several potential venues for your event.
+
+##### What equipment does AB Tech own?
+We have a wide inventory of professional grade equipment available to support a variety of events. Please see <a href="/equipment">Equipment</a> for a general list of equipment we own.  If you do not see a specific type of equipment, contact us and we may be able to suggest alternatives or assist in coordinating rentals from an outside vendor.
+
+##### How are events priced?
+Events are quoted based on the equipment configuration needed and amount of labor we estimate you will require. Last minute requests or changes may incur an additional fee to cover increased support. However, these quotes are estimates; the final invoice may differ depending on the specific event circumstances.
+
+##### How can I request an event?
+Click the <a href="/request">Request Event</a> button up top and fill out the form. We will be in contact with you shortly!
```
Comment (DaAwesomeP) [inline pages/faq.md:102]: Change link to `<a href="{{ '/request/' | relative_url }}">`.

File: pages/faq.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+- Note that events planning to occur during busier weeks (i.e. Orientation, Family Weekend, Carnival, Dancer’s Symposium, etc.)  will need to be requested earlier in advance to ensure equipment and personnel availability.
+
+##### Where is AB Tech able to support events?
+We are able to work at most venues on CMU’s main campus in Pittsburgh, including Mellon College of Science. Most events we support are on-campus, but we also assist with a few select off-campus events each year. Events may be indoors or outdoors, weather permitting.
+- Some types of events favor certain venues over others. Before reserving a venue, please contact us with your event details and we can suggest several potential venues for your event.
+
+##### What equipment does AB Tech own?
+We have a wide inventory of professional grade equipment available to support a variety of events. Please see <a href="/equipment">Equipment</a> for a general list of equipment we own.  If you do not see a specific type of equipment, contact us and we may be able to suggest alternatives or assist in coordinating rentals from an outside vendor.
```
Comment (DaAwesomeP) [inline pages/faq.md:96]: Change link to be `<a href="{{ '/equipment/' | relative_url }}">`.

File: pages/faq.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+Come talk to us! We are present at the FAIR and Winter FAIR - just look for the bright lights and follow the loud music. You can also join the mailing list by hitting the <a href="/join">Join</a> button above. See a person in a tech shirt? Ask them about the next event you can help at and they should hopefully be able to point you in the right direction.
+
+##### How else can I get connected?
+Join our <a href="https://www.facebook.com/cmuabtech" target="_blank">Facebook</a> page and follow us on <a href="https://www.instagram.com/cmuabtech/" target="_blank">Instagram</a> @cmuabtech to see all the cool things we’re doing!
+
+<h2>
+ <a id="Requesting AB Tech">Requesting AB Tech</a>
+</h2>
+##### What events does AB Tech support?
+We support a variety of events on campus! From speaker events and movie nights to full dance shows and concerts, we are available to design the appropriate technical configuration for your event. Some events you may have seen us at in the past include Orientation, Dancer’s Symposium, Lunar Gala, Carnival and Greek Sing. See a list of <a href="/events">Past Events</a>.
```
Comment (DaAwesomeP) [inline pages/faq.md:85]: Change link to be `<a href="{{ '/events/' | relative_url }}">`.

File: pages/faq.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+
+##### Is there a time commitment to being in AB Tech?
+Nope! You are free to sign up for run crew positions and come to setup/strike as your schedule allows. However, depending on the size of the event and run crew position, you may need to attend run crew meetings or prepare paperwork ahead of time. These meetings will be coordinated with the techie in charge as needed.
+
+##### What else do techies do?
+Sure, we work hard, but we also have a lot of fun together. After large events, we often get together for a celebratory meal off campus. We’ve also hosted game nights, karaoke, Fuku runs and ski trips for our members. If you’ve got an idea for a social event, let us know! 
+
+##### How do I get involved?
+Come talk to us! We are present at the FAIR and Winter FAIR - just look for the bright lights and follow the loud music. You can also join the mailing list by hitting the <a href="/join">Join</a> button above. See a person in a tech shirt? Ask them about the next event you can help at and they should hopefully be able to point you in the right direction.
```
Comment (DaAwesomeP) [inline pages/faq.md:76]: Change link to be `<a href="{{ '/join/' | relative_url }}">`.

File: pages/faq.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+ <li>Stage Manager (SM): Responsible for communicating between performers backstage and run crew in house, as well as calling cues and handing off equipment.</li>
+ <li>Assistant Postions: These are assistant positions that are great for beginners to learn about a run crew position. Especially for large events, you may see aLD, aFoH and aTiC positions open where you can work side by side with a more experienced run crew member.</li>
+</ul>
+
+##### Do I need prior experience in event production?
+No, in fact, many of our members have little to no prior experience in event production when starting out. Just about everything can be learned on the job by participating in events and taking run crew positions. 
+
+##### When does AB Tech meet?
+AB Tech holds open weekly meetings for anyone interested in getting involved with upcoming events. Join our mailing list in the <a href="/join">Join</a> form for specific times and locations. We also meet to set up events throughout the week. For a list of upcoming events, please see our <a href="https://abtech.andrew.cmu.edu/tracker/" target="_blank">Tracker</a> site.
```
Comment (DaAwesomeP) [inline pages/faq.md:64]: Change links to be `<a href="{{ '/join/' | relative_url }}">` and `<a href="https://tracker.abtech.org/" target="_blank">`.

File: pages/faq.md [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+Techies can take on several different run crew positions, which are roles that are responsible for specific aspects of an event (for example, lighting). Techies can also help with general setup and strike. Some common run crew positions include:
+<ul>
+ <li>Techie-in-Charge (TiC): Responsible for planning the event, including communicating with the event organizer, finding run crew and coordinating the logistics of the event.</li>
+ <li>Lighting Designer (LD): Responsible for designing where lighting fixtures are placed around the venue and programming the lighting.</li>
+ <li>Front of House Engineer (FoH): Responsible for selecting appropriate speaker systems and mixing inputs so that the show sounds good to the audience.</li>
+ <li>Stage Manager (SM): Responsible for communicating between performers backstage and run crew in house, as well as calling cues and handing off equipment.</li>
+ <li>Assistant Postions: These are assistant positions that are great for beginners to learn about a run crew position. Especially for large events, you may see aLD, aFoH and aTiC positions open where you can work side by side with a more experienced run crew member.</li>
+</ul>
```
Comment (DaAwesomeP) [inline pages/faq.md:58]: It should be possible to do an unordered list just with the `-` syntax like in `equipment.md`. Try removing the `ul` and `li` tags and doing something like:
```md
- Techie-in-Charge (TiC): Responsible for planning the event, including communicating with the event organizer, finding run crew and coordinating the logistics of the event.
- Lighting Designer (LD): Responsible for designing where lighting fixtures are placed around the venue and programming the lighting.
```

File: pages/faq.md [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+##### What exactly does AB Tech do?
+Every year, CMU hosts many events on campus, such as dance shows, concerts, and competitions. These events often require technical support to run smoothly, and that is where AB Tech steps in. We provide the lighting, sound and backstage management needed to make the event a success. Some of our events are as small as providing speakers for a gathering, while others are as large as rigging new structures to hang dozens of lights for a concert. 
+
+##### Where is AB Tech located?
+Our main location is the techroom at the Cohon University Center. To get to techroom from ABP, follow the below directions:
+<ol>
+ <li>From the stairwell, head towards the Gym/Studio A/Keeler. You should pass the glass sculpture on the left.</li>
+ <li>Take a right and follow the hallway up the ramp.</li>
+ <li>Enter the double doors to your left at the top of the ramp.</li>
+</ol>
```
Comment (DaAwesomeP) [inline pages/faq.md:35]: It should be possible to do this without the `ol` and `li` tags. Try doing it like:
```markdown
1. From the stairwell, head towards the Gym/Studio A/Keeler. You should pass the glass sculpture on the left.
2. Take a right and follow the hallway up the ramp.
```

File: _includes/nav.html [hunk 14]
```diff
@@ -22,7 +22,8 @@
...
+		<li><hr class="dropdown-divider"></li>
```
Comment (DaAwesomeP) [inline _includes/nav.html:26]: fix indent

File: pages/faq.md [hunk 15]
```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: FAQ
```
Comment (DaAwesomeP) [inline pages/faq.md:3]: Should the title be `Frequently Asked Questions` or `FAQ`? This is what appears in the tab name in the browser.

**Review by DaAwesomeP — CHANGES_REQUESTED** [review]

File: pages/faq.md (hunk 4 above)
Comment (izhanchang) [reply pages/faq.md:19]: `[## General](#General)` like so?

Comment (DaAwesomeP) [reply pages/faq.md:19]: `[## General](#general)`

File: _includes/nav.html (hunk 14 above)
Comment (izhanchang) [reply _includes/nav.html:26]: Not to be rude but out of ignorance...  where?

Comment (DaAwesomeP) [reply _includes/nav.html:26]: Line 26. You may have mixed tabs and spaces. image

File: pages/faq.md (hunk 4 above)
Comment (izhanchang) [reply pages/faq.md:19]: image
image


I can get markdown working for General and Getting Involved. Physically "requesting AB Tech" looks correct in the dev environment but doesn't scroll like others. On first glance does anything look incorrect about the way it's set up?

Comment (DaAwesomeP) [reply pages/faq.md:19]: The heading is created by doing:
```markdown
## My Heading
```
Jekyll Markdown turns this into HTML: `<h2 id="my-heading">My Heading</h2>`. This markdown (above) creates an anchor of the name `my-heading` which can be linked to like:
```markdown
[Link to jump to my heading](#my-heading)
```
The `[This is my link](#some-location)` Markdown syntax is the same thing as `<a href="#some-location">This is my link</a>` in HTML. Links to anchors begin with `#` and then the anchor name. You can also do other links like `[Google](https://google.com)` which turns into `<a href="https://google.com">Google</a>`.

So for the first image, these are links to jump to other parts of the page (I presume), so you would want:
```markdown
[General](#general)
[Getting Involved](#getting-involved)
[Requesting AB Tech](#requesting-ab-tech)
```
You don't need any HTML for the links. And then later where the heading actually goes you can do:
```markdown
## Requesting AB Tech

Blah blah this is how you request...
```

Comment (DaAwesomeP) [reply pages/faq.md:19]: The simple rules for how a heading is turned into an anchor are:
 - Spaces become dashes
 - Uppercase becomes lowercase

**Pull request merged**